src/components/TopBar/components/UserMenu/UserMenu.tsx

```diff
@@ -55,7 +55,6 @@ export function UserMenu({
 
   return (
     <Menu
-      preferredAlignment="right"
```

Comment:
This should work with default middle, left or right alignment. 

If the popover overlaps the right hand side of the page it should use a different left value.